src/main/java/frc/robot/commands/drivetrain/vision/AprilTagAlign.java

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands.drivetrain.vision;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.Constants.VisionConstants;
+import frc.robot.RobotContainer;
+
+public class AprilTagAlign extends Command {
+  /** Creates a new AprilTagAlign. */
+  private double angle, turn;
+  public AprilTagAlign() {
+    addRequirements(RobotContainer.swerveDrive);
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {}
+
+  // Called every time the scheduler runs while the command is scheduled.
+  @Override
+  public void execute() {
+    angle = RobotContainer.limeSubsystem.getAngleOffset();
+
+    if (RobotContainer.limeSubsystem.getAngleOffset() == 0) {
+      angle = RobotContainer.swerveDrive.getDisplacementToTarget(0);
+    }
```

Comment:
The body of this if statement will probably never be executed since a double will never be exactly zero. You need to do a threshold here if you want this to work.